recipes/json-c/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.12" %}
+
+package:
+    name: json-c
+    version: {{ version }}
+
+source:
+    fn: json-c-{{ version }}.tar.gz
+    url: https://s3.amazonaws.com/json-c_releases/releases/json-c-{{ version }}.tar.gz
+    md5: 3ca4bbb881dfc4017e8021b5e0a8c491
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - autoconf
+        - automake
+        - libtool
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/libjson-c.a  # [unix]
+        - test -f ${PREFIX}/lib/libjson-c.so  # [linux]
+        - test -f ${PREFIX}/lib/libjson-c.dylib  # [osx]
+        - conda inspect linkages -n _test json-c  # [linux]
```

Comment:
Only with latest conda-build. Since that does not work on Windows and I am avoiding OS X we are good. (conda-build is pinned to an old version in the CircleCI due to another conda-bug.)